grafana/version.txt

```diff
@@ -1 +1 @@
-8.3.7-2
+8.4.5-1
```

Comment:
We should make this 8.4.5-2 since there is already a 8.4.5-1 tag.

Reply:
Image was backport. Change: https://github.com/astronomer/ap-vendor/commit/c64bb75a1e82d5409e080e920b6458e5c4c9e539

Reply:
Done